src/components/editor/ListAndDetails.tsx

```diff
@@ -34,25 +52,43 @@ function ListAndDetails({ backgroundColor, list, details, height }: Props) {
                     size={INITIAL_SELECTOR_WIDTH}
                     minSize={MIN_RESIZE_WIDTH}
                 >
-                    <div className="pane-content" style={{ height: heightVal }}>
-                        {list}
+                    <div
+                        className="pane-content"
+                        style={{
+                            height: heightVal,
+                            border: displayBorder ? slateOutline[200] : '',
```

Comment:
I don't think the default/fallback should be an empty string. It should be `undefined` as that is equivalent to not passing the `border` property

Reply:
I try to avoid setting value/attribute explicitly to `undefined`. If that is an acceptable practice in your opinion, it's your codebase. Will correct.